internal/cypress/config.go

```diff
@@ -211,6 +225,18 @@ func ValidateCypressConfiguration(configFile string) error {
 	return nil
 }
 
+func isCypressCfgIgnored() (bool, error) {
+	if _, err := os.Stat(".sauceignore"); err != nil {
```

Comment:
You have to read the location of the .sauceignore file from the project. Because this value is configurable 💡

Reply:
fixed!